bin/rose

```diff
@@ -37,7 +37,9 @@
 if ${ROSE_DEBUG:-false}; then
     set -x
 fi
-. "$(dirname "$0")/../lib/bash/rose_init"
+# shellcheck source=lib/bash/rose_init
```

Comment:
Apparently shellcheck and pip don't play nicely.

Reply:
> Apparently shellcheck and pip don't play nicely.

???

Reply:
Shellcheck objects to the script being sourced because PIP puts it somewhere shellcheck doesn't think to look.